themes/console-home/layouts/tools/list.html

```diff
@@ -66,7 +66,7 @@ <h2 class="title title-1">Developer tools reviews</h2>
         {{- range first 1 $.Site.Data.toolsreviewed.items -}}{{- $reviewedChecked = time (index . "Scheduled for") -}}{{- end -}}
         {{- $updated := $reviewedChecked -}}
             <p class="small">
-				<span>Last updated: {{ $updated.Format "2006-01-02" }}</span>. Read our&nbsp;<a class="link" href="/about/#selection-criteria">selection criteria</a>
+				<span>Last updated: {{ $updated.Format "2006-01-02" }}.</span>&nbsp;Read our&nbsp;<a class="link" href="/about/#selection-criteria">selection criteria</a>.
```

Comment:
Do we need `&nbsp;` rather than just a space?

Reply:
We shouldn't, I've refactored this to keep regular inline flow inside `span` see next commit.